.gitignore

```diff
@@ -12,3 +12,5 @@
 *.patch_*
 *.pdb
 *.target
+Source/Flow/Public/Interfaces/FlowNativeExecutableInterface.h
```

Comment:
We have these files still existing on our version, but the classes are empty (without them some of our assets won't build).

You can skip this file.